### ISSUE TITLE
Use "mouseup" instead of "mousedown"

### ISSUE THREAD

**Issue by brainscript** [sgengler/atom-trello#5]
Currently a few buttons (like the "back" one) use `mousedown` as the click event. That's weird and interferes with Atom's default behavior. In my opinion `mousedown` should only be used to show that an element is active (best practice is obviously using the CSS pseudo class) or for drag and drop behavior.

Is there any particular reason for this decision? If not I would recommend to **change it to `mouseup`**.

**Comment by sgengler** [comment]
@brainscript the reason it was a `mousedown` is that Atom's SelectListView component is set to close the modal when the main search input blurs. It's not really meant to have additional buttons and inputs, so I had to hack it a bit to prevent the input from blurring when clicking the button/checkboxes. Waiting until a `mouseup` to preventDefault() was not enough to prevent the blur.

I just added some additional event listeners to prevent the blur on `mousedown` and moved the functionality to `mouseup` (https://github.com/sgengler/atom-trello/commit/bb703531d4ca05d96a91e3e4a359d3dae6445126). So we should be all good now (Yay!). 

If you update to 0.7.0 you'll be all set :)
